shogun-lib/src/main/java/de/terrestris/shogun/lib/model/security/permission/GroupClassPermission.java

```diff
@@ -2,13 +2,17 @@
 
 import de.terrestris.shogun.lib.model.Group;
 import lombok.*;
+import org.hibernate.envers.AuditTable;
+import org.hibernate.envers.Audited;
 
 import javax.persistence.Entity;
 import javax.persistence.ManyToOne;
 import javax.persistence.Table;
 
 @Entity(name = "groupclasspermissions")
 @Table(schema = "shogun")
+@Audited
+@AuditTable(value = "groupclasspermissions_rev", schema = "shogun_rev")
```

Comment:
do we really need revisions on permissions? i am not sure..

Reply:
Yes, tricky question. Right now the permissions will be instantiated/deleted on entity creation/deletion and it's not even possible to change the permissions directly through any web interface. So - more specifically - the audit table is actually of no use. 
But: I do have a feature branch in progress that implements the appropriate web interfaces to update the permissions for each entity and so the tables *might* become more useful in the near future.

Reply:
ok we will see if there is any practical use for this. nice work!